libusb/os/windows_common.h

```diff
@@ -131,8 +131,9 @@ typedef LONG USBD_STATUS;
 
 #define USBD_SUCCESS(Status)		((USBD_STATUS)(Status) >= 0)
 
+#define USBD_STATUS_STALL_PID		((USBD_STATUS)0xC0000004L)
 #define USBD_STATUS_ENDPOINT_HALTED	((USBD_STATUS)0xC0000030L)
-#define USBD_STATUS_TIMEOUT		((USBD_STATUS)0xC0006000L)
+#define USBD_STATUS_TIMEOUT			((USBD_STATUS)0xC0006000L)
```

Comment:
This change is not necessary.

Reply:
thks, this change is already restored

Reply:
You can also amend the first commit and repush it with git push -f. But no worries, we can do this ourselves when we merge it.

The change would have made sense with 4-chars tab width. But we default to 8-chars tab width.

Reply:
I see, I did not realize that libusb was currently using 8-chars tab width. VSCode defualt to show it with 4-chars. 

Can you tell me how to determine the tab width for a particular project. thks!